README.md

```
@@ -19,7 +23,7 @@ This is the implementation of the "Enigma Messaging System," a modular, multithr
 *   [Running the Application](#running-the-application)
 *   [Running Tests](#running-tests)
 *   [Architecture Overview](#architecture-overview)
-*   [License](#license)
+*   [License](#LICENSE)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
Expected: #license; Actual: #LICENSE

(MD051, link-fragments)